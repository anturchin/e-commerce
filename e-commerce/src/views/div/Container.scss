.container {
    display: flex;
    gap: 30px;
    align-items: center;
}
