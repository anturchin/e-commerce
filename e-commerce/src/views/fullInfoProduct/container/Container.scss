.conteinerInfo {
    display: flex;
    flex-direction: column;
    gap: 0.7rem;

    padding: 1.5rem;
    width: 45rem;
}
