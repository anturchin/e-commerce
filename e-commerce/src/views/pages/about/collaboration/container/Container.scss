.container-colab {
    display: flex;
    padding: 0 2rem;
    align-items: center;
    gap: 45px;
}
