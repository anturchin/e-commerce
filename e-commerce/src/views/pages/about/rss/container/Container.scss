.container-rss {
    display: flex;
    align-items: center;
    padding: 2rem 15rem;
    gap: 60px;
}

@media (max-width: 750px) {
    .container-rss {
        padding: 2rem 13rem;
    }
}

@media (max-width: 650px) {
    .container-rss {
        padding: 2rem 10rem;
    }
}

@media (max-width: 600px) {
    .container-rss {
        padding: 2rem 9rem;
    }
}

@media (max-width: 550px) {
    .container-rss {
        padding: 2rem 7rem;
    }
}

@media (max-width: 500px) {
    .container-rss {
        padding: 2rem 6rem;
    }
}

@media (max-width: 450px) {
    .container-rss {
        padding: 2rem 4rem;
    }
}

@media (max-width: 400px) {
    .container-rss {
        padding: 2rem 2rem;
    }
}

@media (max-width: 360px) {
    .container-rss {
        padding: 2rem 0rem;
    }
}
