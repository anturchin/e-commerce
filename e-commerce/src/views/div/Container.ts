import { View } from '../View';
import './Container.scss';

export class Container extends View {
    constructor() {
        super({ tag: 'div', classNames: ['container', 'conteinerInfo'] });
    }
}
