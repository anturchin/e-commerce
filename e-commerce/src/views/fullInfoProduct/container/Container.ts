import { View } from '../../View';
import './Container.scss';

export class ContainerColumn extends View {
    constructor() {
        super({ tag: 'div', classNames: ['conteinerInfo'] });
    }
}
