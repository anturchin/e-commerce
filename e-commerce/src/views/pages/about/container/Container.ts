import { View } from '../../../View';
import './Container.scss';

export class ContainerUser extends View {
    constructor() {
        super({ tag: 'div', classNames: ['container-user'] });
    }
}
