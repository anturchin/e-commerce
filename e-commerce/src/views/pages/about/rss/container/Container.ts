import { View } from '../../../../View';
import './Container.scss';

export class ContainerRSS extends View {
    constructor() {
        super({ tag: 'div', classNames: ['container-rss'] });
    }
}
