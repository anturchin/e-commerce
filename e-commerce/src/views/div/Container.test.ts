import { describe, expect, test } from '@jest/globals';
import { Container } from './Container';

describe('Container', () => {
    test('should match', () => {
        const container = new Container();
        const renderedContainer = container.getElement();
        expect(renderedContainer.outerHTML).toMatchSnapshot();
    });
});
