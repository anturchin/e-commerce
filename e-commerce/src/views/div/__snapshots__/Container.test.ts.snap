// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container should match 1`] = `"<div class="container conteinerInfo"></div>"`;
